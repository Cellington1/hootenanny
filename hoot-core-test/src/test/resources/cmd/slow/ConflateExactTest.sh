#!/bin/bash
set -e

mkdir -p test-output/cmd/ConlateExactTest
STATS_OUT=test-output/cmd/ConlateExactTest/boston-road-building-out

hoot conflate -D writer.include.debug=true test-files/BostonSubsetRoadBuilding_FromOsm.osm test-files/BostonSubsetRoadBuilding_FromOsm.osm $STATS_OUT.osm --stats > $STATS_OUT

hoot is-match $STATS_OUT.osm test-files/cmd/slow/ConflateExactTest/output.osm
