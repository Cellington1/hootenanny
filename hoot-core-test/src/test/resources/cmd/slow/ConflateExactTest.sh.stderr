Error running conflate:
Unknown geometry type: Relation = restriction
